### ISSUE TITLE
france.tv issue

### ISSUE THREAD

**Issue by yadurhot** [ytdl-org/youtube-dl#28220]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.02.10**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
youtube-dl "https://www.france.tv/france-5/oui-oui-enquetes-au-pays-des-jouets/oui-oui-enquetes-au-pays-des-jouets-saison-2/2294933-l-affaire-des-pneus-creves.html" -f best --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.france.tv/france-5/oui-oui-enquetes-au-pays-des-jouets/oui-oui-enquetes-au-pays-des-jouets-saison-2/2294933-l-affaire-des-pneus-creves.html', '-f', 'best', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.02.10
[debug] Python version 3.7.3 (CPython) - Linux-5.10.11-v7l+-armv7l-with-debian-10.8
[debug] exe versions: ffmpeg 4.1.6-1, ffprobe 4.1.6-1
[debug] Proxy map: {}
[FranceTVSite] 2294933-l-affaire-des-pneus-creves: Downloading webpage
[FranceTV] 8868235f-a545-4ebd-906e-203aaf2835b6: Downloading video JSON
[FranceTV] 8868235f-a545-4ebd-906e-203aaf2835b6: Downloading geo restriction info
[FranceTV] 8868235f-a545-4ebd-906e-203aaf2835b6: Downloading signed hls_v5_os manifest URL
[FranceTV] 8868235f-a545-4ebd-906e-203aaf2835b6: Downloading m3u8 information
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/youtube_dl/YoutubeDL.py", line 806, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/youtube_dl/YoutubeDL.py", line 827, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.7/dist-packages/youtube_dl/extractor/common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.7/dist-packages/youtube_dl/extractor/francetv.py", line 234, in _real_extract
    return self._extract_video(video_id, catalog)
  File "/usr/local/lib/python3.7/dist-packages/youtube_dl/extractor/francetv.py", line 194, in _extract_video
    self._sort_formats(formats)
  File "/usr/local/lib/python3.7/dist-packages/youtube_dl/extractor/common.py", line 1367, in _sort_formats
    raise ExtractorError('No video formats found')
youtube_dl.utils.ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

First time it happens to me

**Comment by eevvoor** [comment]
I confirm this.

**Comment by Vangelis66** [comment]
Using a whitelisted **French IP** and latest `yt-dl` at the time of writing (**2021.05.16**), and **despite the DRM label** having been applied to this issue, the same programme referenced by @yadurhot on Feb 21st 2021 in OP apparently has **no issues now** on May 23rd: 
```
youtube-dl -F "https://www.france.tv/france-5/oui-oui-enquetes-au-pays-des-jouets/oui-oui-enquetes-au-pays-des-jouets-saison-2/2294933-l-affaire-des-pneus-creves.html" => 

[FranceTVSite] 2294933-l-affaire-des-pneus-creves: Downloading webpage
[FranceTV] 8c6ef0db-1d76-4e93-af8e-ba391176c200: Downloading video JSON
[FranceTV] 8c6ef0db-1d76-4e93-af8e-ba391176c200: Downloading geo restriction info
[FranceTV] 8c6ef0db-1d76-4e93-af8e-ba391176c200: Downloading signed hls_v5_os manifest URL
[FranceTV] 8c6ef0db-1d76-4e93-af8e-ba391176c200: Downloading m3u8 information
[FranceTV] 8c6ef0db-1d76-4e93-af8e-ba391176c200: Downloading signed m3u8-download manifest URL
[FranceTV] 8c6ef0db-1d76-4e93-af8e-ba391176c200: Downloading m3u8 information
[info] Available formats for 8c6ef0db-1d76-4e93-af8e-ba391176c200:
format code                             extension  resolution note
hls_v5_os-audio-aacl-96-Audio_Fran?ais  mp4        audio only [fr]
hls_v5_os-522                           mp4        384x216     522k , avc1.42C01E, 25.0fps, video only
hls_v5_os-1105                          mp4        640x360    1105k , avc1.4D401F, 25.0fps, video only
hls_v5_os-1582                          mp4        960x540    1582k , avc1.4D401F, 25.0fps, video only
hls_v5_os-2218                          mp4        1280x720   2218k , avc1.64001F, 25.0fps, video only
m3u8-download-518                       mp4        384x216     518k , avc1.66.30, mp4a.40.2
m3u8-download-1081                      mp4        640x360    1081k , avc1.77.30, mp4a.40.2
m3u8-download-1525                      mp4        960x540    1525k , avc1.77.30, mp4a.40.2
m3u8-download-2113                      mp4        1280x720   2113k , avc1.64001f, mp4a.40.2 (best)
```
So perhaps they're applying DRM when a programme is "new" (recently broadcast), but lifting DRM as it becomes "stale"... Just my 2p... 😜
